### PR TITLE
JAMES-2223 NPE in MessageContantExtractor when no content dispostion

### PR DESCRIPTION
**Pull request by quynhn**

### CODE REVIEW

File: server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature [hunk 1]
```diff
@@ -419,4 +419,12 @@ Feature: GetMessages method
     When "alice@domain.tld" ask for messages "m1"
     Then no error is returned
     And the list should contain 1 message
-    And the isForwarded property of the message is "false"
\ No newline at end of file
+    And the isForwarded property of the message is "false"
+
+  Scenario: Message with inlined attachment but without content disposition should
```
Comment (rouazana) [inline server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/resources/cucumber/GetMessages.feature]: should what?

File: server/container/util-java8/src/main/java/org/apache/james/util/mime/MessageContentExtractor.java [hunk 2]
```diff
@@ -173,7 +173,9 @@ private boolean isNotAttachment(Entity part) {
     }
 
     private boolean isInlinedWithoutCid(Entity part) {
-        return part.getDispositionType().equals(MimeMessage.INLINE) && part.getHeader().getField(CONTENT_ID) == null;
+        return part.getDispositionType() != null
```
Comment (chibenwa) [inline server/container/util-java8/src/main/java/org/apache/james/util/mime/MessageContentExtractor.java]: Can't we rely on Objects.equals(part.getDispositionType(), MimeMessage.INLINE) ?

**Comment by quynhn** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by aduprat** [comment]
test this please

**Comment by quynhn** [comment]
test this please

**Comment by quynhn** [comment]
> CassandraSetMessagesMethodTest>SetMessagesMethodTest.attachmentsAndBodyShouldBeRetrievedWhenChainingSetMessagesAndGetMessagesWithTextBodyAndHtmlAttachment:3900 » ConditionTimeout

**Comment by quynhn** [comment]
test this please

**Comment by quynhn** [comment]
>Errors: 
[ERROR]   HBaseMailboxSessionMapperFactoryTest.testCreateMailboxMapper » NoClassDefFound
[ERROR]   HBaseMailboxSessionMapperFactoryTest.testCreateMessageMapper » NoClassDefFound
[ERROR]   HBaseMailboxSessionMapperFactoryTest.testCreateSubscriptionMapper » NoClassDefFound
[ERROR]   HBaseMailboxSessionMapperFactoryTest.testGetModSeqProvider » NoClassDefFound C...
[ERROR]   HBaseMailboxSessionMapperFactoryTest.testGetUidProvider » ExceptionInInitializer

**Comment by quynhn** [comment]
test this please

**Pull request merged**